apps/journeys-admin/src/components/TemplateView/AccountCheckDialog/AccountCheckDialog.tsx

```diff
@@ -0,0 +1,39 @@
+import Button from '@mui/material/Button'
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+import { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
```

Comment:
Use next instead of the react one. React one causing hydration error issue

```suggestion
import { useTranslation } from 'next-i18next'
```